es-core/src/Window.cpp

```diff
@@ -335,8 +335,8 @@ void Window::setHelpPrompts(const std::vector<HelpPrompt>& prompts, const HelpSt
 				// yes, it has!
 
 				// can we combine? (dpad only)
-				if((it->first == "up/down" && addPrompts.at(mappedTo->second).first == "left/right") ||
-					(it->first == "left/right" && addPrompts.at(mappedTo->second).first == "up/down"))
+				if((strcmp(it->first, "up/down") == 0 && addPrompts.at(mappedTo->second).first == "left/right") ||
```

Comment:
Question: shouldn't the addPrompts comparison also be with strcmp then in these two lines?

addPrompts.at(mappedTo->second).first == "left/right")